src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+  const values = Object.values(robot);
+
+  for (let i = 0; i < values.length; i++) {
+    for (let j = 0; j < values.length; j++) {
+      if (j !== i && values[i] === values[j]) {
+        return null;
+      }
+    }
+  }
+
+  for (const i in robot) {
+    inversedRobot[robot[i]] = i;
+  }
```

Comment:
How about using for of loop with Object.entries combined in it?

```suggestion
for (const [key, value] of Object.entries(robot)) {
  // Check if value already exist as a property, if true, return null
  // Inverse => inversedRobot[value] = key
}